src/main.js

```diff
@@ -27,19 +37,39 @@
     }
 
     attachNonPreventedListener(document, "click", (e) => {
+        // ignore ctrl / meta clicks (may be opening in new tab or window)
+        if (e.ctrlKey || e.metaKey) {
```

Comment:
You don’t need this check. Click event is not triggered when you open a context menu for a link.

Reply:
To be fair I mirrored what I think the old browserstate ajaxify script was doing. Checking for ctrl click at least makes a difference.

Haven't touched javascript in a while, after playing around more with the script I think a better approach might be to have a custom event and event handler specifically for when a link was clicked.  That way the click event could be skipped by the user.

Reply:
Looked around, found this:
```javascript
	function shouldNavigate(event) {
		return !event.defaultPrevented && event.button === 0 && !(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
	};
	```
	from react.js's internals for handling clicks. I also tried this locally, metaKey might be the context menu which might not matter, but all the other keys do get grabbed by the script.